jb-yoda/src/main/java/life/jukebox/JBWatcher.java

```diff
@@ -0,0 +1,22 @@
+import java.util.concurrent.*;
+
+public class JBWatcher implements Runnable{
+  private ScheduledFuture<?> watcherHandle;
```

Comment:
watcherHandle doesn't communicate this is the handle to the scheduled task that runs this watcher - maybe taskHandle?

Reply:
fixed